packages/rn-tester/js/examples/TurboModule/SampleTurboModuleExample.js

```diff
@@ -168,7 +168,8 @@ const styles = StyleSheet.create({
     justifyContent: 'space-between',
   },
   value: {
-    fontFamily: Platform.OS === 'ios' ? 'Menlo' : 'monospace',
+    fontFamily:
```

Comment:
Add a macOS tag?

Reply:
Yes we need a tag here or this will potentially get wiped out with the 0.67 merge

Reply:
why settle there? why menlo? can't it be san francisco? :)

Reply:
Tag added.